debezium-server/debezium-server-eventhubs/src/main/java/io/debezium/server/eventhubs/EventHubsChangeConsumer.java

```diff
@@ -125,57 +125,80 @@ public void handleBatch(List<ChangeEvent<Object, Object>> records,
             op.setMaximumSizeInBytes(maxBatchSize);
         }
 
-        EventDataBatch batch = producer.createBatch(op);
+        for (int i = 0; i < records.size();) {
```

Comment:
I think the variable `i` should be renamed. The reason is that this is no longer an ordinary `for` so usign only traditiional var name is very misleading and hard to reason about the code.

Reply:
Seems fair, I'll adjust.